### DISCUSSION TITLE
Addon for Heimdall?

### DISCUSSION THREAD

**Discussion by Viking-1** [alexjustesen/speedtest-tracker#425] [Ideas]
The old tracker from henry whitaker had a plugin for heimdall. I tested it with yours but it's not working. Should it?
And if not, will there be one in the future?

**Comment by alexjustesen** [comment]
@Viking-1 can you send me references to the Heimdall plugin? Pretty sure it uses the same APIs as the other dashboards I'm tracking in #81

**Comment by Viking-1** [comment]
![grafik](https://user-images.githubusercontent.com/125771058/221349640-d8e7beea-e9d5-449a-872a-00040af98db8.png)
Thats happening in Heimdall. If I use https instead of http, there is no error but also nothing else (no data).
On Heimdall.site you will redirected to: https://github.com/henrywhitaker3/Speedtest-Tracker
![grafik](https://user-images.githubusercontent.com/125771058/221349733-f7a130ca-18b3-4d7e-8a2f-5add8fe905a9.png)

I've changed the password, user and mail in the tracker settings. Could this be a problem?

**Reply by alexjustesen** [reply]
yep that's to be expected, the api endpoints aren't in place yet.
